src/types/Todo.ts

```diff
@@ -0,0 +1,9 @@
+import { User } from './User';
+
+export interface Todo {
+  id: number;
+  title: string;
+  completed: boolean;
+  userId: number;
+  user?: User | null;
```

Comment:
`?` - it says ur value(`user`) can be undefined 
```suggestion
  user: User | null;
```